content/source/docs/enterprise/api/variables.html.md

```diff
@@ -22,16 +22,25 @@ This POST endpoint requires a JSON object with the following properties as a req
 Properties without a default value are required.
 
 
-Key path                    | Type   | Default | Description
-----------------------------|--------|---------|------------
-`data.type`                 | string |         | Must be `"vars"`.
-`data.attributes.key`       | string |         | The name of the variable.
-`data.attributes.value`     | string |         | The value of the variable.
-`data.attributes.category`  | string |         | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.
-`data.attributes.hcl`       | bool   | `false` | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.
-`data.attributes.sensitive` | bool   | `false` | Whether the value is sensitive. If true then the variable is written once and not visible thereafter.
-`filter.workspace.name`     | string |         | The name of the workspace that owns the variable.
-`filter.organization.name`  | string |         | The name of the organization that owns the workspace.
+Key path                                    | Type   | Default | Description
+--------------------------------------------|--------|---------|------------
+`data.type`                                 | string |         | Must be `"vars"`.
+`data.attributes.key`                       | string |         | The name of the variable.
+`data.attributes.value`                     | string |         | The value of the variable.
+`data.attributes.category`                  | string |         | Whether this is a Terraform or environment variable. Valid values are `"terraform"` or `"env"`.
+`data.attributes.hcl`                       | bool   | `false` | Whether to evaluate the value of the variable as a string of HCL code. Has no effect for environment variables.
+`data.attributes.sensitive`                 | bool   | `false` | Whether the value is sensitive. If true then the variable is written once and not visible thereafter.
+`data.relationships.workspace.data.type`    | string |         | Must be `"workspaces"`.
+`data.relationships.workspace.data.id`      | string |         | The ID of the workspace that owns the variable.
+`data.relationships.organization.data.type` | string |         | Must be `"organizations"`.
+`data.relationships.organization.data.id`   | string |         | The ID (which is the name) of the organization that owns the workspace.
+
+**Deprecation warning**: The following properties are deprecated and will be removed from feature versions of the API!
```

Comment:
This looks like a great add. I'd replace "feature" with "future", and note something like "`filter` properties have been replaced by `relationships`.

Reply:
That's indeed a typo, will update that and add a word about the `filter` <-> `relationships` change as well. Thought that would be obvious, doesn't hurt to write it down 👍 Thx!